front/src/modules/pathfinding/hooks/usePathfinding.ts

```diff
@@ -230,7 +241,38 @@ export const usePathfinding = (
             pathfindingResult.failed_status === 'pathfinding_not_found' &&
             pathfindingResult.error_type === 'incompatible_constraints';
 
-          if (pathfindingResult.status === 'success' || incompatibleConstraintsCheck) {
+          const isInvalidPathItems =
```

Comment:
Maybe it's a better name
```suggestion
          const hasInvalidPathItems =
```